smart-contract-verifier-http/tests/settings.rs

```diff
@@ -1,23 +1,28 @@
 use pretty_assertions::assert_eq;
 use smart_contract_verifier_http::Settings;
 
-// For linux systems we assume that all os specific values
-// are filled with defaults, so no need to rewrite them.
-#[cfg(target_os = "linux")]
-fn rewrite_os_specific_example_settings(
-    _example_settings: &mut Settings,
-    _default_settings: &Settings,
-) {
-}
-
-// For other systems we just use the values from default settings.
-#[cfg(not(target_os = "linux"))]
 fn rewrite_os_specific_example_settings(
```

Comment:
```suggestion
fn rewrite_system_specific_example_settings(
```
That is not _os_specific_ rewritings, as, for example, `max_threads` are not specific to any OS